src/test/java/net/minecraftforge/debug/item/ItemAttributeModifierTest.java

```diff
@@ -16,6 +16,7 @@
  * License along with this library; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
+
```

Comment:
Again with the whitespace change.

Reply:
Again from my [previous comment](https://github.com/MinecraftForge/MinecraftForge/pull/7655/commits/13d20fd3e44ef2b575663bd455ff39898b0ad905#r580661234), this is the work of the `updateLicenses` task.